### PR TITLE
List  Dragonglass (DRGL)

### PR DESCRIPTION
**Pull request by aejontargaryen**
*    Official project URL: https://zirtysperzys.org
*    Official block explorer URL: http://zirtysperzys.info
*    Arbitation: DRGL wallets support Payment ID to assist with 3rd party audits in case of a dispute (CryptoNote technology)

### CODE REVIEW

File: src/main/java/bisq/asset/coins/DRAGONGLASS.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Bisq. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package bisq.asset.coins;
+
+import bisq.asset.Coin;
+import bisq.asset.RegexAddressValidator;
+
+public class DRAGONGLASS extends Coin {
```
Comment (blabno) [inline src/main/java/bisq/asset/coins/DRAGONGLASS.java]: DRAGONGLASS -> Dragonglass

File: src/test/java/bisq/asset/coins/DRAGONGLASSTest.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+ * along with Bisq. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package bisq.asset.coins;
+
+import bisq.asset.AbstractAssetTest;
+
+import org.junit.Test;
+
+public class DRAGONGLASSTest extends AbstractAssetTest {
```
Comment (blabno) [inline src/test/java/bisq/asset/coins/DRAGONGLASSTest.java]: DRAGONGLASSTest -> DragonglassTest

**Comment by blabno** [comment]
Also the commit message does not conform to our guide.

**Comment by aejontargaryen** [comment]
10-4.
I'll do new pr for simplicity

**Pull request closed without merging**